code/FormSchemaController.php

```diff
@@ -35,9 +34,11 @@ abstract class FormSchemaController extends AdminController
 
     protected function init()
     {
-        $this->beforeExtending('onInit', function () {
-            $this->prepareTinyMce();
-        });
+        // Set the members html editor config
+        $currentUser = Security::getCurrentUser();
+        if ($currentUser) {
+            HTMLEditorConfig::set_active_identifier($currentUser->getHTMLEditorConfigForCMS());
+        }
```

Comment:
Since only this tiny part is needed here now, there's no need to have the extra method.